src/disasm.cpp

```diff
@@ -670,7 +664,7 @@ static void jl_dump_asm_internal(
 
     // Set up Subtarget and Disassembler
     std::unique_ptr<MCSubtargetInfo>
-        STI(TheTarget->createMCSubtargetInfo(TripleName, MCPU, Features.getString()));
+        STI(TheTarget->createMCSubtargetInfo(TripleName, cpu, features));
```

Comment:
I rather think we should just be using `"", ""` here (like the defaults for llvm-objdump).

Although if we have `object != NULL`, we could be very fancy, and use
`TheTriple = object->makeTriple(); features = object->getFeatures();`

Reply:
No that's what's causing https://bugs.llvm.org/show_bug.cgi?id=30525

Reply:
And I also don't want to rely on the feature set of the object since I think it should be allowed to embed instructions in the code that is not generally available as long as it's protected by feature checks.